helpers/tcpdump.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import os
+import time
+import subprocess
+import signal
+import argparse
+import shutil
+
+class Logger():
+    def __run_tcpdump(self, ethname) -> None:
+        """
+        Run `tcpdump` before the test if `-s` (--save-tcpdump) option is used.
+        Save result in a <name>.pcap file, where <name> is name of test.
+        """
+        path = f"/var/tcpdump/{datetime.date.today()}"
+        file_name = datetime.datetime.now().strftime('%H:%M:%S')
+
+        if not os.path.isdir(path):
+            os.makedirs(path)
+        self.__tcpdump = subprocess.Popen(
+            [
+                "tcpdump",
+                "-U",
+                "-i",
+                f"{ethname}",
+                "-w",
+                f"{path}/{file_name}.pcap",
+            ],
+            shell=False,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+
+    def __clear_tcpdump_files(self, directory, cur_time, time_to_clear_dump) -> None:
+        for _f in os.listdir(directory):
+            f = os.path.join(directory, _f)
+            t = os.path.getctime(f)
+            if cur_time - t > time_to_clear_dump:
+                os.remove(f)
+
+    def __clear_tcpdump(self, cur_time, time_to_clear_dump) -> None:
```

Comment:
We can use `-C` (file size) and `-W` (file count) options https://www.tcpdump.org/manpages/tcpdump.1.html and then this method is not needed